platform-modules/ekstep-framework/src/main/java/org/ekstep/framework/mgr/impl/FrameworkManagerImpl.java

```diff
@@ -104,12 +89,14 @@ public Response createFramework(Map<String, Object> request, String channelId) t
 	@Override
 	public Response readFramework(String frameworkId, List<String> returnCategories) throws Exception {
 		Response response = new Response();
-		if (Platform.config.hasPath("framework.es.sync")) {
-			if (Platform.config.getBoolean("framework.es.sync")) {
+		if (Platform.config.hasPath("framework.cassandra.sync")) {
```

Comment:
Remove this configuration (framework.cassandra.sync). We should always push the data to cassandra when it is published.

Reply:
This property was used in Read Framework API Only. Removed. Now Read Framework API will try to fetch data from cassandra, if not present in cassandra, fetch it from neo4j.